broker/lib/utils.py

```diff
@@ -56,18 +56,16 @@ def parse_domain_options(params):
         return [d.strip().lower() for d in domains]
 
 
-def handle_domain_updates(params, instance):
-    domain_names = parse_domain_options(params)
-    no_domain_updates = True
-    if domain_names is not None:
+def handle_domain_updates(params, instance) -> list[str]:
+    updated_domain_names = parse_domain_options(params) or []
+    if len(updated_domain_names) > 0:
```

Comment:
`parse_domain_options` returns `None` if there are no domain options passed in - I think using `None` as our no update case would be a little better here, if only so we have one fewer place where we change the value